src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleasePublishingStatusRepository.cs

```diff
@@ -49,10 +50,10 @@ public ReleasePublishingStatusRepository(IPublisherTableStorageService publisher
             }
 
             var query = new TableQuery<ReleasePublishingStatus>().Where(filter);
-            return _publisherTableStorageService.ExecuteQueryAsync(PublisherReleaseStatusTableName, query);
+            return _publisherTableStorageService.ExecuteQuery(PublisherReleaseStatusTableName, query);
```

Comment:
I think we agreed that in cases like this we would change to add `await` in rather than returning a Task that's not awaited, so I've made this change. Thanks.